Default.sublime-commands

```diff
@@ -126,7 +129,7 @@
 
     // Code Folding
     { "caption": "Code Folding: Fold", "command": "fold" },
-    { "caption": "Code Folding: Fold All", "command": "fold_all" },
+    { "caption": "Code Folding: Fold All", "command": "fold_by_level", "args": {"level": 1} },
```

Comment:
I don't think I want to make this change.

Instead adding an explicit "Fold by Level 1" is preferable.

```
{ "caption": "Code Folding: by Level 1", "command": "fold_by_level", "args": {"level": 1} },
```

Reply:
This corresponds to the "official" content of "Default.sublime-commands", lines 136-138:
```
	{ "caption": "Code Folding: Fold", "command": "fold" },
	{ "caption": "Code Folding: Fold All", "command": "fold_by_level", "args": {"level": 1} },
	{ "caption": "Code Folding: Unfold", "command": "unfold" },
```

Reply:
I am aware of that, but it doesn't mean to apply it under all circumstances in this repo.

Reply:
done!